dashboard/db/migrate/20170222192220_remove_href_from_external.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveHrefFromExternal < ActiveRecord::Migration[5.0]
+  def up
+    External.all.each{|level| level.update!(properties: level.properties.except('href'))}
+  end
```

Comment:
Ah, sorry, I meant we want to be able to run the migration both ways even though we're only modifying data in the `up` migration.

Some migrations raise `ActiveRecord::IrreversibleMigration` if you run them backwards — I wasn't sure if that was also true for migrations with only an `up` and not a `down` method.